docs/sources/reference/components/beyla/beyla.ebpf.md

```diff
@@ -84,17 +85,20 @@ You can use the following blocks with `beyla.ebpf`:
 | `filters` > [`network`][network filters]                               | Configures filtering of network attributes.                                                        | no       |
 | [`metrics`][metrics]                                                   | Configures which metrics Beyla exposes.                                                            | no       |
 | `metrics` > [`network`][network metrics]                               | Configures network metrics options for Beyla.                                                      | no       |
+| `traces`                                                       | Configures trace collection and sampling options for the component. | no       |
```

Comment:
should this say `for all services instrumented by the component` instead of `for the component`?